packages/core/types/index.js

```diff
@@ -334,29 +373,29 @@ export type Transformer = {
     asset: MutableAsset,
     resolve: ResolveFn,
     options: ParcelOptions
-  }) => Async<Config | void>,
+  }) => Async<ThirdPartyConfig | void>,
```

Comment:
Can we add `loadConfig` and `rehydrateConfig` to the `Transformer` type here?

Reply:
Oh yeah, meant to do that.